admin/includes/stylesheet.css

```diff
@@ -835,6 +835,16 @@ input.faint {
 
 /* End of css buttons */
 
+/* Additions for help */
+.navbar {
+  margin-bottom: 8px;

```

Comment:
The addition of this margin has pointed out that adding the help button has forced a vertical loss of space for every page on which the button appears. 
Perhaps the HTML where it is rendered can be reworked to prevent forcing all the page content to be shoved down just so the button can be shown?

Reply:
You have to push things down otherwise the button will overwrite on page content.

Reply:
If this is a big concern we could make it a link next to support, but I think making it more visible is better.   Let's try it with the blue color from the details button on the orders screen so it stands out from the gray in the navbar above it.